applications/DEMApplication/DEM_application.cpp

```diff
@@ -278,6 +278,7 @@ KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(LOCAL_AUX_ANGULAR_VELOCITY)
 // ******************* Quaternion Integration END *******************
 
 // FORCE AND MOMENTUM
+KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(CONTACT_IMPULSE)
```

Comment:
why are you adding this in DEM?

Reply:
I also thought it could be added in the coupling application, but I did not say anything because I thought the variable could be useful in the future, when we develop the 'hard sphere strategy' which uses impulses. I don't really know before starting the implementation...